index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <div id="container-logo"><a id="myLogo" >My Logo</a></div>     
+        <div id="container-menu"><a id="menu"><img src="css/img/hamburger.svg"></a></div> 
+    </nav>
+
+    <section id="headline">
+        <div id="container-text-icons">
+        <div id="text-presentation">
+           <h1>I'm Bianca. <br>
+                Glad to see 
+                you!
+            </h1>
+            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <h4>LET'S CONNECT</h4>
+        </div>
+         <div id="container-icons">
+            <img id="twitter" src="css/img/twitter-icon.png" alt="Twitter">
+            <img id="link"  src="css/img/Linkedin-icon.png" alt="LinkedIn">
+            <img id="m"  src="css/img/M-icon.png" alt="">
+            <img id="github" src="css/img/github-icon.png" alt="GitHub">
+            <img id="paz" src="css/img/paz-icon.png" alt="">
+        </div>

```

Comment:
- [ ] I would kindly suggest that all the social media images should be wrapped inside of a `ul` with an `li` and each `li` with an `a`
   - The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).
      - Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using href.

E.g :point_down: 
```HTML
<ul>
  <li><a href="#">social media image</a></li>
  <li><a href="#">social media image</a></li>
  <li><a href="#">social media image</a></li>
  <li><a href="#">social media image</a></li>
  <li><a href="#">social media image</a></li>
</ul>
```